src/redist/redist.csproj

```diff
@@ -41,125 +41,94 @@
     </Content>
   </ItemGroup>
 
-  <Target Name="PublishAspNetRuntimePackageStore"
-          AfterTargets="CrossgenPublishDir">
+  <Target Name="PublishAspNetRuntimePackageStore" AfterTargets="CrossgenPublishDir">
```

Comment:
I'm guessing VS did all this reformatting, but it's making it hard to review. Can you revert?